### PR TITLE
doc: Add Amazon EKS to the list of implementations

### PR DESCRIPTION
**Pull request by liwenwu-amazon**
**What type of PR is this?**

Add one of the following kinds:

/kind documentation

**What this PR does / why we need it**:

AWS Gateway API Controller is a new implementation of the Gateway API that may be of interest to the wider audience of the Gateway API project. 

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: site-src/implementations.md [hunk 1]
```diff
@@ -27,11 +28,13 @@ Implementors and integrators of Gateway API are encouraged to update this docume
 - [STUNner][21] (beta)
 - [Traefik][13] (alpha)
 
+
```
Comment (shaneutt) [inline site-src/implementations.md:31]:
```suggestion
```

File: site-src/implementations.md [hunk 2]
```diff
@@ -27,11 +28,13 @@ Implementors and integrators of Gateway API are encouraged to update this docume
...
 ## Integration Status
 - [Flagger][14] (public preview)
 - [cert-manager][15] (alpha)
 - [argo-rollouts][22] (alpha)
 
+
```
Comment (shaneutt) [inline site-src/implementations.md:37]:
```suggestion
```

File: site-src/implementations.md [hunk 3]
```diff
@@ -54,6 +57,8 @@ Implementors and integrators of Gateway API are encouraged to update this docume
 [20]:#big-ip-kubernetes-gateway
 [21]:#stunner
 [22]:#argo-rollouts
+[23]:#amazon-kubernetes-service
```
Comment (shaneutt) [inline site-src/implementations.md:60]:
```suggestion
[23]:#amazon-elastic-kubernetes-service
```

File: site-src/implementations.md [hunk 4]
```diff
@@ -54,6 +57,8 @@ Implementors and integrators of Gateway API are encouraged to update this docume
...
+
```
Comment (shaneutt) [inline site-src/implementations.md:61]:
```suggestion
```

**Comment by robscott** [comment]
Thanks @liwenwu-amazon!

/lgtm
/approve

File: site-src/implementations.md [hunk 5]
```diff
@@ -66,6 +71,15 @@ Documentation can be found at [EPIC Application & API Gateway Service][epic]
...
 
+### Amazon Elastic Kubernetes Service
+
+[Amazon Elastic Kubernetes Service (EKS)][eks] is a managed service that you can use to run Kubernetes on AWS without needing to install, operate, and maintain your own Kubernetes control plane or nodes. EKS's implementation of the Gateway API is through [AWS Gateway API Controller][eks-gateway] which provisions [Amazon VPC Lattice][vpc-lattice] Resources for gateway(s), HTTPRoute(s) in EKS clusters.
+
+
+[eks]:https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html
+[eks-gateway]:https://github.com/aws/aws-application-networking-k8s
+[vpc-lattice]:https://aws.amazon.com/vpc/lattice/
+
```
Comment (shaneutt) [inline site-src/implementations.md:82]: :+1:

File: site-src/implementations.md [hunk 6]
```diff
@@ -293,6 +307,7 @@ Traefik is currently working on implementing UDP, and ReferenceGrant. Status upd
 [traefik]:https://traefik.io
 [traefik-1]:https://doc.traefik.io/traefik/routing/providers/kubernetes-gateway/
 
+
```
Comment (shaneutt) [inline site-src/implementations.md:310]:
```suggestion
```

**Review by shaneutt — CHANGES_REQUESTED** [review]
Couple small things, then looks good! Glad to have you join the implementations page :partying_face:

**Comment by shaneutt** [comment]
This auto-merged through my request for changes, which included a fix for the link for EKS so I've created a follow-up https://github.com/kubernetes-sigs/gateway-api/pull/1921

**Pull request merged**